appengine/src/test/java/module/DBIProvider.java

```diff
@@ -14,10 +14,19 @@
  * limitations under the License.
  */
 
-package com.google.googleinterns.gscribe.resources.io.exception;
+package module;
 
-public class AuthenticationException extends RuntimeException {
-    public AuthenticationException(String message) {
-        super(message);
+import org.skife.jdbi.v2.DBI;
+
+public class DBIProvider {
```

Comment:
Is this required? There are two DBIProvider in src/test

Reply:
Forgot to remove while changing branches. This provider will be deleted.